dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -487,6 +487,12 @@
      */
     public static final String REGISTRY_RETRY_PERIOD_KEY = "retry.period";
 
+    /**
+     * Period of checkZk for providers retry interval
+     */
+    public static final String  REGISTRY_CHECK_ZK_PERIOD_KEY  = "checkzk.period";
```

Comment:
pls. reuse `REGISTRY_RECONNECT_PERIOD_KEY` instead of introducing a new constants.

Reply:
ok.... i will solve it